.github/ISSUE_TEMPLATE/project_story_template.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Project Story Template
+about: Project story template for tracking on the infra project board 
+title: ''
+labels: card
+assignees: ''
+---
+
+**Summary**
+
+> Summary the purpose of this story
+
+**Impact**
+
+> State which strategic or tactical goal this impacts (e.g. S2 or T1).
+> Refer to the infra charter [here](https://drive.google.com/open?id=1GFwzH3BGjkI-a8bDxREg9oJHcmab3wbw9m4xAfb-bBk)
+
+**Tasks**
+- Task1
+- Task2
+- Task3
```

Comment:
Random idea: Maybe make these checkboxes instead of bullets? Via `- [ ] Task1`.

Reply:
like it